site/src/atomics/flex.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Flex Atomics
+description: Flex related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Flex Atomics
+
+Flex atomics are useful when composing layouts and patterns containing other components.
+
+| cssproperty       | value                                                                               | screensize |
+| ----------------- | ----------------------------------------------------------------------------------- | ---------- |
+| `flex-direction`  | `column`, `row-reverse`, `column-reverse`                                           | `tablet`   |
+| `justify-content` | `flex-start`, `flex-end`, `center`, `space-around`, `space-between`, `space-evenly` | `tablet`   |
+| `align-items`     | `flex-start`, `flex-end`, `center`, `baseline`, `stretch`                           | `tablet`   |
+| `align-self`      | `flex-start`, `flex-end`, `center`, `baseline`, `stretch`                           | `tablet`   |
+| `flex-wrap`       | `flex-wrap-wrap`, `flex-wrap-wrap-reverse`                                          | `tablet`   |
+| `align-content`   | `flex-start`, `flex-end`, `space-around`, `space-between`, `stretch`                | `tablet`   |
+| `flex-grow`       | `0`, `1`                                                                            | `tablet`   |
+| `flex-shrink`     | `0`, `1`                                                                            | `tablet`   |
+
+## Usage
```

Comment:
1. Delete second example? It looks a bit off to my eye.
2. Now that flex direction row is in main, use that instead of row-reverse?
2. Add explanation? Suggestion: `The following example uses a combination of mobile-first flex atomics to achieve a responsive card layout.`

Reply:
updated